ts/features/bonus/bpd/screens/iban/insertion/IbanInsertionComponent.tsx

```diff
@@ -0,0 +1,77 @@
+import { Input, Item, View } from "native-base";
+import * as React from "react";
+import { useState } from "react";
+import { SafeAreaView } from "react-native";
+import { Iban } from "../../../../../../../definitions/backend/Iban";
+import { Body } from "../../../../../../components/core/typography/Body";
+import { H1 } from "../../../../../../components/core/typography/H1";
+import { H5 } from "../../../../../../components/core/typography/H5";
+import { IOStyles } from "../../../../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import I18n from "../../../../../../i18n";
+import { FooterTwoButtons } from "../../../../bonusVacanze/components/markdown/FooterTwoButtons";
+
+type Props = {
+  onBack: () => void;
+  onIbanConfirm: (iban: Iban) => void;
+  onContinue: () => void;
+};
+
+// https://en.wikipedia.org/wiki/International_Bank_Account_Number
+// Italian Iban max length: 27 (sample: IT60X0542811101000000123456)
+const IbanMaxLength = 34;
+
+const loadLocales = () => ({
+  headerTitle: I18n.t("bonus.bpd.title"),
+  title: I18n.t("bonus.bpd.iban.insertion.title"),
+  body1: I18n.t("bonus.bpd.iban.insertion.body1"),
+  body2: I18n.t("bonus.bpd.iban.insertion.body2"),
+  ibanDescription: I18n.t("bonus.bpd.iban.iban"),
+  skip: I18n.t("bonus.bpd.iban.skip")
+});
+
+export const IbanInsertionComponent: React.FunctionComponent<Props> = props => {
+  const [iban, setIban] = useState("");
+  const isInvalidIban = iban.length > 0 && Iban.decode(iban).isLeft();
```

Comment:
```suggestion
  const isInvalidIban = Iban.decode(iban).isLeft();
```
`Iban.decode("") -> left`

Reply:
I did this to avoid that when the user enter in the screen the first time (`iban.length===0`) immediately sees the error graphic without having started entering text yet.
<img src=https://user-images.githubusercontent.com/26501317/95320762-3dd09e00-089a-11eb-9948-afbc210158b1.png width=300 />
What you think?